cpanfile

```diff
@@ -14,13 +14,13 @@ requires 'Selenium::Chrome';
 requires 'Selenium::Waiter';
 requires 'Selenium::Remote::WDKeys';
 requires 'Digest::file';
+requires 'YAML::Tiny';
```

Comment:
Somehow missed that. Actually `Test::Assert` is runtime dependency too. Could you please add it in same PR? Otherwise I will handle it.

Reply:
I will take care.

Reply:
Thanks!